scripts/Muon/widget_helper.py

```diff
@@ -0,0 +1,33 @@
+from __future__ import (absolute_import, division, print_function)
+from PyQt4 import QtCore, QtGui
+
+
+def setName(table,row,name):
+    text = QtGui.QTableWidgetItem((name))
+    text.setFlags(QtCore.Qt.ItemIsEnabled)
+    table.setItem(row,0, text)
+
+
+def createComboTable(table,row,options):
```

Comment:
My final comment about the last 3 is that the name to me implies that a table is being created when in fact they modify and existing table. How about something like `addComboBoxToTable` for the name here?